### DISCUSSION TITLE
Don’t ask OTP code next X days

### DISCUSSION THREAD

**Discussion by lusnyaktah** [keycloak/keycloak#35124] [Ideas]
Does keycloak has a plan to add such an option in the future “Don’t ask code next X days”. ?
Like in MS
![image](https://github.com/user-attachments/assets/bf5f89f3-5393-4cae-94bc-ff50db1504d5)

**Comment by erwin1** [comment]
I have no idea about Keycloak plans, but there is this plugin https://github.com/wouterh-dev/keycloak-spi-trusted-device that does exactly that and works really well.

**Reply by mabartos** [reply]
cc: @keycloak/core

**Reply by lusnyaktah** [reply]
Thanks @erwin1 for your reply
